### PR TITLE
Remove duplicity and make tests verbose

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this?**
/kind cleanup

**What does this PR do / why we need it**:
This PR cleans some duplicity from the code, fixes typo and makes tests verbose.

**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [ ] Update changelog

- [ ] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: tests/integration/cmd_pref_config_test.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/tidwall/gjson"
 )
 
-const promtMessageSubString = "Help odo improve by allowing it to collect usage data."
+const promptMessageSubString = "Help odo improve by allowing it to collect usage data."
 
 var _ = Describe("odo preference and config command tests", func() {
 	// TODO: A neater way to provide odo path. Currently we assume \
```
Comment (valaparthvi) [inline tests/integration/cmd_pref_config_test.go:16]: What does this TODO mean? Has it already been fixed?

**Review by rnapoles-rh — APPROVED** [review]
/lgtm

**Pull request merged**